### PR TITLE
[bug 1083982] Show unapproved articles in KB dashboard

### PR DESCRIPTION
**Pull request by rehandalal**
r?

### CODE REVIEW

File: kitsune/dashboards/readouts.py [hunk 1]
```diff
@@ -230,32 +230,37 @@ def kb_overview_rows(mode=None, max=None, locale=None, product=None):
...
+            data['stale'] = data['expiry_date'] < datetime.now()
 
         # Check L10N status
-        unapproved_revs = d.revisions.filter(
-            reviewed=None, id__gt=d.current_revision.id)[:1]
+        if d.current_revision:
+            unapproved_revs = d.revisions.filter(
+                reviewed=None, id__gt=d.current_revision.id)[:1]
+        else:
+            unapproved_revs = None
```
Comment (mythmon) [inline kitsune/dashboards/readouts.py]: Shouldn't this be `d.revisions.all()` ?

**Comment by rehandalal** [comment]
fixed ^

**Comment by rehandalal** [comment]
now with test ^

**Comment by mythmon** [comment]
Cool, r+

**Pull request merged**